### PR TITLE
Chemistry fixes

### PR DESCRIPTION
**Pull request by tortellinitony**
Adds solidification: uranium, gold, silver, bananium, glass, metal. Also fixed grinding. Also fixed mint toxin, why the hell not.

### CODE REVIEW

**Comment by ArcLumin** [comment]
Muh solids

**Comment by ArcLumin** [comment]
Also, code's good

File: code/modules/reagents/chemistry/machinery/reagentgrinder.dm [hunk 1]
```diff
@@ -411,10 +411,14 @@
...
-				O.reagents.trans_to(beaker, amount)
-				if(!O.reagents.total_volume)
-						remove_object(O)
+				var/allowed = get_allowed_by_id(O)
+				for (var/r_id in allowed)
+						var/space = beaker.reagents.maximum_volume - beaker.reagents.total_volume
+						var/amount = allowed[r_id]
+						beaker.reagents.add_reagent(r_id,min(amount, space))
+						if (space < amount)
+								break
```
Comment (Carbonhell) [inline code/modules/reagents/chemistry/machinery/reagentgrinder.dm:423]: funny indentation?

Comment (Carbonhell) [reply code/modules/reagents/chemistry/machinery/reagentgrinder.dm:423]: no nvm

**Review by Carbonhell — APPROVED** [review]

**Pull request merged**